integration_tests/backend/test_reads.py

```diff
@@ -17,6 +17,9 @@
 
 
 class TestBackend:
+    V2_GET_WORKFLOWS_TEMPLATE = "/api/v2/workflows"
+
+    GET_WORKFLOWS_TEMPLATE = "/api/workflows"
```

Comment:
Should we keep the v1 route around if we are moving over to v2?

Who else is using this route? Is it just our SDK?

Reply:
I'm using it for testing the endpoint (`assert len(v1_resp) == len(v2_resp)`). I think in the future we would want to remove it but I think while it is not deprecated it may be nice using it.

Reply:
Actually on second thought I've decided to remove the dependency on V1.